R/get_contact_matrix.R

```diff
@@ -0,0 +1,48 @@
+#' Create a contact matrix
+#'
+#' Function that loads and processes a contact matrix for use in the CoMo model
+#' ode solver. This function carries out a dimension change on the input
+#' contact matrix (from 16x16 to 21x21) to allow for the larger number of
+#' population categories provided by the population data
+#' (0-100+ years in 5 year intervals) compared to the contact data (0-80 in 5
+#' year intervals). This is done by repeating entries in the contact data
+#' corresponding to the oldest group (75-80 years) for the last population data
+#' age ranges (i.e. all 5 year categories for 80+ years).
+#'
+#' @param country String representing the country for which the contact matrix
+#' will be created. Naming conventions for countries must be in line with those
+#' used in input excel file.
+#' @param contact_type A named list of 16x16 matrices. Names are countries for
+#' which contact matrices are available. Each contact matrix represents the
+#' contact between different age groups in a given social setting e.g.
+#' contact_work gives contact matrix for the work place.
+#' @param n_5yr_agegroups_popstruc Number of 5 year age groups in the population
+#'  structure data. these ranges start with 0-4yrs and continue with 5 year gaps
+#'  up until the last category of 100+ years old. Default value set to 21.
+#'
+#' @return A square contact matrix for the specified country with dimension
```

Comment:
Can we say what the rows versus the columns mean? (We say that it's asymmetric somewhere, but does (say) a row mean the age group we are considering and the column mean the other age group or is it vice versa?)

Reply:
Done in package description - rows refer to age group of respondents (people at risk) and columns refer age group of people "contacted" by people at risk.